src/cairo_project.toml

```diff
@@ -1,2 +0,0 @@
-[crate_roots]
-openzeppelin = "."
```

Comment:
Ah this old toml. We stopped needing this when we integrated scarb, right?

Reply:
Yes, and it conflicts with the new LSP version.